agent/pkg/singularity/singularity.go

```diff
@@ -62,10 +62,10 @@ type SingularityContainer struct {
 
 // SingularityClient implements ContainerRuntime.
 type SingularityClient struct {
-	log        *logrus.Entry
-	opts       options.SingularityOptions
+	Log        *logrus.Entry
```

Comment:
Note that if you had put the podman.go in the same package as this one you wouldn't have had to change the case to make them accessible.

Might make sense to rationalize the package and struct names after Bradley's PR lands.

Reply:
Sounds good. The package names container & containers are in use already. Perhaps container_runtimes?